docs/network_disruption/prio.md

```diff
@@ -117,7 +117,7 @@ spec:
 #### (Step 1) Add a fourth band
 
 <p align="center"><kbd>
-    <img src="../docs/img/network_prio/1-4.png" height=220 width=650 />
+    <img src="../../docs/img/network_prio/1-4.png" height=220 width=650 />
```

Comment:
```suggestion
    <img src="../img/network_prio/1-4.png" height=220 width=650 />
```
instead of leaving and rejoining `docs`?

Reply:
Nope, it does not work 😞 : 
![image](https://user-images.githubusercontent.com/8859832/220659256-bb5fea5b-cedc-4dca-a5c3-ca8745b59b84.png)